content/docs/plugins.md

```diff
@@ -85,8 +85,54 @@ for any technical questions/answers and the announcement your new plugin.
 
 A lexer plugin is a kind of extension compared to a normal plugin,   
 i.e. in addition to all the functions already mentioned, additional ones have to be exported and also an XML file defining the styles has to be generated.  
-  
-Notepad++ (Npp) currently supports only the [ILexer4](https://www.scintilla.org/ScintillaDoc.html#LexerObjects) interface.  
+
+#### Notepad++ v8.4 and newer
+
+Notepad++ has transitioned to the [ILexer5](https://www.scintilla.org/LexillaDoc.html) interface as of Notepad++ v8.4.
+A lexer plugin needs to define all methods of this interface to ensure a smooth interaction.  
+
+The lexer functions to be exported by the plugin are
+- GetLexerCount  
+- GetLexerName  
+- CreateLexer
+- LexerNameFromID
+
+The [`SCI_SETILEXER`](https://www.scintilla.org/ScintillaDoc.html#SCI_SETILEXER) message is used to set the current lexer to your defined ILexer5 object.
```

Comment:
Calling `SCI_SETILEXER` is the job of Notepad++ to load external lexer plugin, but it's not necessary to use `SCI_SETILEXER` in the external lexer plugin.
I suggest to remove this line to avoid the confusion.